presto-parser/src/main/java/com/facebook/presto/sql/parser/AstBuilder.java

```diff
@@ -190,10 +191,12 @@
 {
     private int parameterPosition;
     private final ParsingOptions parsingOptions;
+    private final ParsingWarningCollector warningCollector;
```

Comment:
This is unused.

Reply:
Yeah, but it is nice to have in this class. Don't feel strong if we should keep it though.